OutOfSchool/OutOfSchool.DataAccess/Repository/EntityRepository.cs

```diff
@@ -89,5 +89,58 @@ public async Task<T> Update(T entity)
             await this.dbContext.SaveChangesAsync();
             return entity;
         }
+
+        /// <inheritdoc/>
+        public async Task<int> Count(Expression<Func<T, bool>> where = null)
+        {
+            if (where == null)
+            {
+                return await dbSet.CountAsync().ConfigureAwait(false);
+            }
+            else
+            {
+                return await dbSet.Where(where).CountAsync().ConfigureAwait(false);
+            }
+        }
+
+        /// <inheritdoc/>
+        public IQueryable<T> Get<TOrderKey>(int skip = 0, int take = 0, string includeProperties = "", Expression<Func<T, bool>> where = null, Expression<Func<T, TOrderKey>> orderBy = null, bool ascending = true)
```

Comment:
this method definition is long and not all parameters are visible without scrolling
it would be better to have method parameters on new line each